app/services/doc_auth/acuant/requests/liveness_request.rb

```diff
@@ -0,0 +1,48 @@
+module DocAuth
+  module Acuant
+    module Requests
+      class LivenessRequest < DocAuth::Acuant::Request
+        attr_reader :image
+
+        def initialize(config:, image:)
+          super(config: config)
+          @image = image
+        end
+
+        def headers
+          super().merge 'Content-Type' => 'application/json'
```

Comment:
I know it's unchanged, but for my personal style I'd flip the parens here:

```suggestion
          super.merge('Content-Type' => 'application/json')
```